website/source/docs/agent/options.html.md

```diff
@@ -592,6 +592,8 @@ default will automatically work with some tooling.
         ACLs are enabled. This token may be provided later using the [agent token API](/api/agent.html#update-acl-tokens)
         on each server. If the `replication` token is set in the config. This token must have at least "read" permissions
```

Comment:
Not your thing but the sentence: "If the `replication` token is set in the config." is jank/typo here, can you remove it while touching this?